packages/hebao_v1/contracts/modules/security/InheritanceModule.sol

```diff
@@ -74,7 +74,6 @@ abstract contract InheritanceModule is SecurityModule
         txAwareHashNotAllowed()
         onlyFromWalletOrOwnerWhenUnlocked(wallet)
     {
-        require(controllerCache.walletRegistry.isWalletRegistered(_inheritor), "NOT_A_WALLET");
```

Comment:
Not really necessary to enforce this onchain. We can enforce rules for inheritors in the app. If someone manually wants to set an arbitrary address as an inheritor that's also fine, but they'll have to do that by directly interacting with the contracts, and so if they can do that they can also do all other operations they want to do manually themselves as well.